captum/optim/_param/image/transforms.py

```diff
@@ -459,3 +466,98 @@ def __init__(self, warp: bool = False) -> None:
     def forward(self, x: torch.Tensor) -> torch.Tensor:
         assert x.dim() == 4
         return nchannels_to_rgb(x, self.warp)
+
+
+class RandomCrop(nn.Module):
+    """
+        Randomly crop out a specific size from an NCHW image tensor.
+    ​
+        Args:
+            crop_size (int, sequence, int): The desired cropped output size.
+    """
+
+    def __init__(
+        self,
+        crop_size: Union[Tuple[int, int], Tuple[int]],
```

Comment:
Keeping in line with how `center_crop` and `CenterCrop` work already, the `__init__` function should be this instead:

```
    def __init__(
        self,
        crop_size: IntSeqOrIntType,
    ) -> None:
        super().__init__()
        crop_size = [crop_size] * 2 if not hasattr(crop_size, "__iter__") else crop_size
        crop_size = list(crop_size) * 2 if len(crop_size) == 1 else crop_size
        crop_size = cast(Union[List[int], Tuple[int, int]], crop_size)
        assert len(crop_size) == 2
        self.crop_size = crop_size
```

The documentation already reflects this change, so no change should be needed there.